README.md

```diff
@@ -20,21 +19,95 @@ The following properties will be synchronized:
 * Prerequisites
 * Fallthrough rule
 
-Quick setup
------------
+## Quick setup
 
 1. Install with `npm`
 
-        npm install --save
+   ```
+   npm install
+   ```
 
 2. Run the script, passing in the project key, source and destination environments, and API token:
 
-        node index.js --project-key PROJECT_KEY --source-env SOURCE_ENVIRONMENT_KEY --destination-env DEST_ENVIRONMENT_KEY --api-token API_TOKEN
-   
+   ```
+   ./sync-ld-flags --api-token API_TOKEN \
+     --project-key PROJECT_KEY \
+     --source-env SOURCE_ENVIRONMENT_KEY \
+     --destination-env DEST_ENVIRONMENT_KEY
+   ```
+
    **Note:** The environment key can be found in Account Settings within the LaunchDarkly application immediately under the name of the desired environment.
    
    Optionally pass a host override:
 
-        node index.js -p PROJECT_KEY -s SOURCE_ENVIRONMENT_KEY -d DEST_ENVIRONMENT_KEY -t API_TOKEN -H https://your-launch-darkly-deploy.com
+   ```
+   ./sync-ld-flags --api-token API_TOKEN \
+     -H https://your-launch-darkly-deploy.com
```

Comment:
Will the user need single quotes around the URL here?

Reply:
nope, should be ok without it